test/providers/oci_images.test.js

```diff
@@ -0,0 +1,23 @@
+import { expect } from "chai";
+import fs from 'fs';
+import sinon from 'sinon'
+import { generateImageSBOM, parseImageRef } from '../../src/oci_image/utils.js';
+
+let clock;
+suite('testing the OCI image data provider', () => {
```

Comment:
It would be good if you added some more tests. Specially for the corner cases like invalid platforms or image strings.
Also you could implement specific unit tests to validate the individual functionality. 
Given our current situation, consider this optional.